### PR TITLE
doc: move chainer readme content into sphinx project

### PR DESCRIPTION
**Pull request by eslesar-aws**
*Issue #, if available:*

Moved content from /src/sagemaker/chainer/README.rst into sphinx project using_chainer.rst file.

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [x] I have read the [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [x] I have updated the [changelog](https://github.com/aws/sagemaker-python-sdk/blob/master/CHANGELOG.rst) with a description of my changes (if appropriate)
- [x] I have updated any necessary [documentation](https://github.com/aws/sagemaker-python-sdk/blob/master/README.rst) (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: SUCCEEDED
* [Build Logs](https://9dzx6wxcy9.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=592fa90f-412a-4410-bc64-cec7080339d2%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by laurenyu** [comment]
with the newly-merged #705, the changelog is now generated

**Comment by eslesar-aws** [comment]
@laurenyu Does this mean I should remove my manual addition to the changelog?

**Comment by laurenyu** [comment]
@eslesar-aws yep. when you merge master, it'll probably complain about conflicts with the changelog - just take whatever master's version of the changelog is

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: SUCCEEDED
* [Build Logs](https://9dzx6wxcy9.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=d2c8c572-9ed6-4204-8ebb-f140b64683e9%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,19 @@
...
+* doc-fix: Move workflow and PyTorch readme content into sphinx project
+* doc-fix: Fix broken links in using_tensorflow.rst
+* doc-fix: Update inference pipelines documentation
+* doc-fix: Move Chainer readme content into sphinx project
+
+1.18.5
+======
+=======
 ## 1.18.5
+>>>>>>> 840ba48edfe655eef71d90faac8644f8f7b53820:CHANGELOG.md
```
Comment (laurenyu) [inline CHANGELOG.md]: can you remove these changes as well? I think you can do a `git checkout master -- CHANGELOG.md` to reset this file

**Comment by laurenyu** [comment]
@eslesar-aws hmm the merge is still looking a little weird. can you `git fetch` the latest changes to master, and then try the `git checkout` again?

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: SUCCEEDED
* [Build Logs](https://9dzx6wxcy9.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=9faee864-58c7-4107-b097-97b79aedb8c2%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by eslesar-aws** [comment]
I'm just going to kill this PR and start over. It was a very easy change, so it's not worth trying to figure out the weirdness in the changelog

**Pull request closed without merging**